frontend/app/src/components/settings/general/language/LanguageSelectorItem.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="d-flex align-center">
+    <div class="ml-1 d-flex align-center">
+      <div
+        v-for="(country, index) in countries"
+        :key="country"
+        class="d-flex align-center"
+      >
+        <span v-if="index > 0" class="px-1">/</span>
+        <span :class="!dense ? $style['big-flag'] : null">
```

Comment:
can you give a try to useCssModule here? from `vue`

Reply:
sure 🤙